service_crategen/src/commands/generate/codegen/xml_payload_parser.rs

```diff
@@ -81,9 +81,9 @@ fn payload_body_parser(payload_type: ShapeType,
     };
 
     let response_body = match payload_type {
-        ShapeType::Blob if !streaming => format!("body"),
+        ShapeType::Blob if !streaming => "body".to_string(),
         ShapeType::Blob if streaming => format!("{}(response.body)", mutate_type_name_for_streaming(payload_member)),
-        _ => format!("String::from_utf8_lossy(&body).into_owned()"),
+        _ => "String::from_utf8_lossy(&body)".to_string(),
```

Comment:
I think this needs to have the `.into_owned()` inside the string.  See failing S3 compilation.